src/pages/doc/user-guide/glossary.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import DocWithJsx from '../../../templates/doc-jsx'
+import useGlossary from '../../../utils/front/glossary'
+
+const Glossary: React.FC = () => {
+  const { contents } = useGlossary()
+
+  return (
+    <DocWithJsx slug="/doc/user-guide/glossary" headings={[]}>
+      <h1>Glossary</h1>
+      {contents.map(
```

Comment:
ideally this would be case-insensitive sorted alphabetically by name

Reply:
Terms are now sorted!